.spectral.yaml

```diff
@@ -26,6 +26,20 @@ rules:
           -  multipart/form-data
           -  application/octet-stream
 
+  must-return-content-types:
+    description: "All enpoint bodies MUST return the header with (one of) Content-Type: application/json, image/png, application/octet-stream"
+    message: "{{description}}"
```

Comment:
In the message could we list the not accepted content type used?

Reply:
That would make the description too long. We can probably add this to documentation but I think it's redundant. We only mention what are accepted. If it's not in the list, it is not accepted.